devtools/create_project/scripts/scummvm-sludge.natvis

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Debug visualizers for sludge engine types for Visual Studio 2012 and up.
+
+    To use, copy this file into Documents\Visual Studio 20xx\Visualizers.
+-->
+
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+	
+	<!-- Variables -->
+	
+	<Type Name="Sludge::Variable">
+		<DisplayString Condition="varType==SVT_INT">{varData.intValue}</DisplayString>
+		<DisplayString Condition="varType==SVT_FILE">File: {varData.intValue}</DisplayString>
+		<DisplayString Condition="varType==SVT_FUNC">Func: {Sludge::allUserFunc[varData.intValue]} (varData.intValue)</DisplayString>
+		<DisplayString Condition="varType==SVT_FUNC">Func: Unknown user function (varData.intValue)</DisplayString>
```

Comment:
There are two identical conditions with different results here. Is this correct?

Reply:
Good catch, but it's on purpose. Should the first one fail, the second will be used instead. Or so I interpret this.